run.go

```diff
@@ -94,6 +94,15 @@ func IsScreenClearedEveryFrame() bool {
 	return ui.IsScreenClearedEveryFrame()
 }
 
+// SetScreenFilterEnabled enables/disables the use of the "screen" filter Ebiten uses.
+//
+// The "screen" filter is a box filter from game to display resolution.
+//
+// If disabled, nearest-neighbor filtering will be used for scaling instead.
```

Comment:
Add these:

* The default state is true.
* SetScreenFilterEnabled is concurrent-safe.

Reply:
Also added that it takes effect at next Draw call - so calling it during Draw is one frame delayed.